OracleSiebel/OEL/build/multi-stage/cache/dockerfile

```diff
@@ -0,0 +1,81 @@
+# Globals
+
+ARG VERSION=19.03
+ARG BUILD_DATE=2019-04-30
+
+
+FROM oraclelinux:latest as temp
+ARG INSTALLSITE=installers
+ARG VERSION
+
+COPY oraInst.loc getBase.sh getUpdate.sh cache/cache_base.rsp cache/cache_install_base.sh cache/cache_update.rsp.templ cache/cache_install_update.sh siebelkeystore.jks /config/
+
+# Install Siebel Base
+
+ENV LANG=en_US.UTF-8
+
+RUN mkdir -p -ma+rwx /siebel/cache /siebel/oraInventory /stage \
+&& groupadd -g 1000 siebel && useradd -m -g siebel -G siebel -u 1000 siebel \
+&& yum -y install wget tcsh ksh \
+&& cd /mnt \
+&& date \
+&& echo Downloading installer \
+&& bash /config/getBase.sh ${INSTALLSITE} \
+&& date \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/unzip \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/runInstaller \
+&& cd .. \
+&& echo Installing Siebel ... \
+&& bash /config/cache_install_base.sh \
+&& echo Waiting for deployment of web archive ... \
+&& sleep 30 \
```

Comment:
What does `/config/cache_install_base.sh` do that means it may not have finished by the next step in the script?

Reply:
It deploys a tomcat server. As part of that job, a war file is deployed. If we don't give some time, the container is killed before deployment is complete, thus truncating the required content.